### PR TITLE
Bugfix, update version number, changelog and docs

### PR DESCRIPTION
**Pull request by kindly**
Changes:

* Fix issue where scope table is not used when using full paths.
* Correct table names in docs
* Update version number and changelog

### CODE REVIEW

**Comment by kindly** [comment]
@duncandewhurst 

This was an issue missed when changing the scope table from having `_summary` at the end.

**Review by duncandewhurst — APPROVED** [review]

**Comment by duncandewhurst** [comment]
@jpmckinney whilst preparing examples to share with helpdesk analysts, I noticed a bug, which David has fixed in this PR. I've also corrected some incorrect table names in the docs and updated the version number and changelog so this is now ready to release on PyPI.

Please could you review and merge the PR and then tag and push the release? Thanks!

File: ocdskingfishercolab/__init__.py [hunk 1]
```diff
@@ -429,7 +430,7 @@ def get_table_and_path(field, scope_table):
         table = "release_summary"
 
         for num, table_canditate in enumerate(table_candidates):
-            if scope_table == table_canditate:
+            if scope_table[:-8] == table_canditate:
```
Comment (jpmckinney) [inline ocdskingfishercolab/__init__.py:433]: Please correct all occurrences to `table_candidate`.

Comment (duncandewhurst) [reply ocdskingfishercolab/__init__.py]: Done in https://github.com/open-contracting/kingfisher-colab/pull/58/commits/408ec27a28af273db1358326af7f353d81f452d1

**Comment by jpmckinney** [comment]
Tagged and pushed. Will be available at https://pypi.org/project/ocdskingfishercolab/

**Comment by duncandewhurst** [comment]
Thanks!

**Pull request merged**